pkgs/development/perl-modules/Bio-BigFile/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, perl, buildPerlModule, fetchpatch, fetchurl, fetchFromGitHub, kent,
+  BioPerl, IOString, libpng, libmysqlclient, openssl }:
+# This version of bio::DB::bigfile is suited for ENSEMBL VEP
+# See https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile
+
+# Only kent 335 works with Bio-BigFile. The kent package on nix must be slightly changed
+# 1. building utils fails (see patche below)
+# 2. we need headers (PR submitted for kent package)
+let
+  kent335 = kent.overrideAttrs (old: rec {
+    pname = "kent";
+    version = "335";
+
+    src = fetchFromGitHub {
+      owner = "ucscGenomeBrowser";
+      repo = pname;
+      rev = "v${version}_base";
+      sha256 = "1455dwzpaq4hyhcqj3fpwgq5a39kp46qarfbr6ms6l2lz583r083";
+    };
+    patches = [
+      ./kent-utils.patch
```

Comment:
Could you add a comment to this patch? Is this a manual backport of a patch that was merged upstream?